cmd/core-command/res/configuration.toml

```diff
@@ -1,5 +1,11 @@
 [Writable]
 LogLevel = 'INFO'
+  [Writable.InsecureSecrets]
+    [Writable.InsecureSecrets.DB]
+    path = "redisdb"
```

Comment:
other configuration keys are using PascalCase.  Should we also use this format for the secrets config?

Reply:
No, this path has to match to value used for the DB type, which is all lowercase